docs/user/getting-started.md

```diff
@@ -166,7 +166,18 @@ $ FC_CLI_BINARY=flightctl-linux-amd64
 Download the `flightctl` binary to your machine:
 
 ```console
-$ curl -LO https://github.com/flightctl/flightctl/releases/download/latest/${FC_CLI_BINARY}
+$ curl -LO https://github.com/flightctl/flightctl/releases/latest/download/${FC_CLI_BINARY}
```

Comment:
Note the original URL is valid, too. You are changing what's being downloaded:

`curl -LO https://github.com/flightctl/flightctl/releases/download/latest/${FC_CLI_BINARY}` downloads the latest release in the list (v0.4.0-rc3)

`curl -LO https://github.com/flightctl/flightctl/releases/latest/download/${FC_CLI_BINARY}` downloads the release marked "latest" (v0.3.0)

We need to decide what we want here.